Packs/CrowdStrikeFalconStreamingV2/Integrations/CrowdStrikeFalconStreamingV2/CrowdStrikeFalconStreamingV2.py

```diff
@@ -570,7 +570,9 @@ def main():
     params: Dict = demisto.params()
     base_url: str = params.get('base_url', '')
     client_id: str = params.get('client_id', '')
```

Comment:
Client_id is like the identifier to client secret - like username and password. So change that as well, please. 
So both should be the same credentials set